wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -200,6 +207,21 @@ def available_workers(self) -> Dict[int, _Worker]:
             _id: w for _id, w in self._workers.items() if _id not in self.busy_workers
         }
 
+    def _init_wandb_run(self) -> SdkRun:
+        """Controls resume or init logic for a scheduler wandb run."""
+        if self._kwargs.get("run_id"):  # resume
```

Comment:
Since this is run by launch, the resume="allow" should resume runs with the same run id, so as long as the `WANDB_RUN_ID` is set by during launch, this is unncessary